APIs/UpcomingChanges.qmd

```diff
@@ -0,0 +1,100 @@
+# Upcoming changes
+
+This is the list of upcoming changes to Catalog APIs.
+
+## OpenSearch interface error handling update
+
+Please be informed that the **OData API** interface will be updated soon. The exact date of the update will be announced in a separate release note no later than 5 days before the scheduled change.
```

Comment:
Can we also specify where the release will be announced? On the same page or dataspace news page

Reply:
added in:
https://github.com/eu-cdse/documentation/pull/289/commits/598e95767a13f11292a7be1b22ed402e1c9fd267